ngraph/frontend/paddlepaddle/src/op/argmax.cpp

```diff
@@ -2,8 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "argmax.hpp"
```

Comment:
Why did you remove `argmax.hpp` I didn't find any forward declarations? Maybe I miss something.

Reply:
All forward declaration are now moved to #

https://github.com/zhangYiIntel/openvino/blob/81bbfeb0e80159b54af09b19d1d2afd100bca3b5/ngraph/frontend/paddlepaddle/src/op_table.cpp#L16

Reply:
ok, got it.